crates/lakekeeper/src/service/authz/mod.rs

```diff
@@ -214,6 +214,10 @@ pub enum CatalogProjectAction {
     ListRoles,
     SearchRoles,
     GetEndpointStatistics,
+    ModifyTaskQueueConfig,
+    GetTaskQueueConfig,
+    GetAllTasks,
+    ControlAllTasks,
```

Comment:
We should make these actions more specific (GetProjectTask) as otherwise we would have a breaking change when we extend the endpoint to return more then just project tasks:
1. If we would just return the additional tasks, we would have a Semantic breaking change as different entities are returned for the same call
2. If we would not return the additional tasks, we would need to introduce an new permission, one for "only project", and one for all.

In general its a good idea to start with as fine-grained permission as possible.